docs/porting-guides/6.x.x.md

```diff
@@ -30,9 +30,10 @@ The following Ansible components have been removed from the `arista.avd` Ansible
 | Component type | Name | Replacement |
 | -------------- | ---- | ----------- |
 | Filter | `arista.avd.xxx` | |
-| Action plugin/Module | `arista.avd.xxx` | |
+| Action plugin/Module | `arista.avd.configlet_build_config` | The internal `arista.avd.configlet_build_config` module is no longer used by AVD. The plugin is released as open source, so it can be copied and reused according to the license and copyright. |
```

Comment:
same lets do this in a separate PR (with the right title) that will also remove `inventory_to_containers`